src/amo/components/SearchResult/index.js

```diff
@@ -132,7 +124,11 @@ export class SearchResultBase extends React.Component<InternalProps> {
     }
 
     return (
-      <div className="SearchResult-wrapper">
+      <Link
+        href={addon ? this.getAddonLink(addon, addonInstallSource) : '#'}
```

Comment:
I believe that we still want to use the `to` attribute when we have an `addon` as it doesn't force a page refresh, which is what using the `href` attribute will do. So I think the logic needs to be: If we have an `addon` use `to={this.getAddonLink(addon, addonInstallSource)}`, otherwise use `href="#"`.

Reply:
okay, understand